nixos/modules/services/networking/legit.nix

```
@@ -0,0 +1,183 @@
+{ config, lib, pkgs }:
```

```suggestion
{ config, lib, pkgs, ... }:
```

Here the ellipsis are needed.